test/cmd/images.sh

```diff
@@ -57,6 +57,9 @@ os::cmd::try_until_success 'oc get imagestreamtags wildfly:latest'
 os::cmd::expect_success_and_text "oc get imageStreams wildfly --template='{{ index .metadata.annotations \"openshift.io/image.dockerRepositoryCheck\"}}'" '[0-9]{4}\-[0-9]{2}\-[0-9]{2}' # expect a date like YYYY-MM-DD
 os::cmd::expect_success_and_text 'oc get istag' 'wildfly'
 
+# we have an istag here, let's see if the debug command works properly on it
+os::cmd::expect_success_and_text "oc debug istag/wildfly:latest -o yaml" 'image: openshift/wildfly-100-centos7'
```

Comment:
Honestly, I think it's worth it. @smarterclayton can override that if he wants, but the locality of it is worth more than whatever two minute time span we lose with it, since it's most likely going to be hidden by parallel tasks anyway.